File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/colorselector/ColorSelectionListener.java
package net.kdt.pojavlaunch.colorselector;

/**
 * 色が選択されたときのリスナーインターフェース。
 */
public interface ColorSelectionListener {
    /**
     * ColorSelectorによって色が選択されたときに呼び出されます。
     * @param color 選択された色
     */
    void onColorSelected(int color);
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/ui/view/colorselector/ColorSelectionListener.kt
package com.arata.yukarilauncher.ui.view.colorselector

/** 色が選択された際のコールバックインターフェース。 */
interface ColorSelectionListener {
    /** 色が選択されたときに呼び出される。 @param color 選択された色（ARGB） */
    fun onColorSelected(color: Int)
}
